tests/test_server.py

```diff
@@ -537,7 +538,7 @@ async def test_train_core_success_with(
     assert os.path.exists(os.path.join(model_path, "fingerprint.json"))
 
 
-async def test_train_with_retrieval_events_success(
```

Comment:
Why did you edit the original test name? The name is now a duplicate of the test below that gets called in this test body.

Reply:
Oops, this is a mistake. Fixed it.